serde/de.py

```diff
@@ -646,21 +646,21 @@ def {{func}}(data, reuse_instances):
   # create fake dict so we can reuse the normal render function
   fake_dict = {"fake_key":data}
 
-  error = "Exhausted all types"
+  errors = ["Exhausted all types"]
```

Comment:
I think without the "Exhausted all types" message and an extra space in front of the primitives error message it would look even nicer.
```
Can not deserialize 'not-ip-or-uuid' of type str into Union[int, bool].
Reasons:
 input is not of type int
 input is not of type bool
```